lib/resolvers.js

```diff
@@ -64,16 +64,18 @@ const wrapResolvers = function (bind, resolvers = {}) {
         wrapped[name] = {};
       }
 
-      for (const [resolverName, func] of Object.entries(value)) {
+      for (const [resolverName, funcOrPrimitive] of Object.entries(value)) {
```

Comment:
I don't think it is necessary to rename the variable.

Reply:
no prob. I'll change it back.